src/PubMed/eutils_sql_save.jl

```diff
@@ -1,37 +1,29 @@
 using ..DBUtils
 using SQLite
 using MySQL
+using EzXML
+using DataFrames
 
 """
 pubmed_save_efetch(efetch_dict, conn)
 
 Save the results (dictionary) of an entrez-pubmed fetch to the input database.
 """
-function save_efetch!(conn::Union{MySQL.Connection, SQLite.DB}, efetch_dict, verbose=false)
+function save_efetch!(conn::Union{MySQL.Connection, SQLite.DB}, articles::EzXML.Node, verbose=false, cleanup=false)
 
     #Decide type of article based on structrure of efetch
-    articles = nothing
-    if haskey(efetch_dict, "PubmedArticle")
-        TypeArticle = PubMedArticle
-        articles = efetch_dict["PubmedArticle"]
-    else
-        println(efetch_dict)
+
+    if nodename(articles) != "PubmedArticleSet"
+        println(articles)
         error("Save efetch is only supported for PubMed searches")
     end
 
-    println("Saving " , length(articles) ,  " articles to database")
-    for xml_article in articles
-        article = TypeArticle(xml_article)
-        # println("=============Article=====================")
-        # println(article)
-        db_insert!(conn, article, verbose)
+    println("Saving " , countelements(articles) ,  " articles to database")
 
-        #-------MeshHeadingList
-        mesh_heading_list = MeshHeadingList(xml_article)
-        db_insert!(conn, article.pmid, mesh_heading_list, verbose)
-    end
+    parsed = pubmed_to_dfs(articles)
+
+    db_insert!(conn, parsed, cleanup=true)
 
-    conn
 end
```

Comment:
I feel like save_efetch! is no longer needed. 

Instead there is a `PubMed.parsexml` or `Pubmed.parse` that corresponds to pubmed_to_dfs. Then in the processes  - we have only one process search_and_parse. The user can manually call `db_insert!(conn, parsed, cleanup=true)` we just give an example. 
For large queries do you see and advantage on saving every 10_000/30_000 instead of at the end?